CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ All notable changes to this project will be documented in this file based on the
 
 ### Added
 - Elastica\QueryBuilder\DSL\Query::geo_distance
+- Elastica\Aggregation\GeoCentroid
```

Comment:
If you already edit the changelog, you could also add #1150 at the end (see change examples below)